src/app/helptext/system/bootenv.ts

```diff
@@ -35,9 +33,7 @@ export const helptext_system_bootenv = {
 
   rename_name_placeholder: T("Name"),
   rename_name_tooltip: T(
```

Comment:
This helptext can be discarded in favor of reusing the "create_name_tooltip" - the help text should be identical.